addons/pos_mercado_pago/models/mercado_pago_pos_request.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import requests
+
+_logger = logging.getLogger(__name__)
+
+
+REQUEST_TIMEOUT = 10
+MERCADO_PAGO_API_ENDPOINT = 'https://api.mercadopago.com'
+
+class MercadoPagoPosRequest:
+    def __init__(self, mp_bearer_token):
+        self.mercado_pago_bearer_token = mp_bearer_token
+
+    def _call_mercado_pago(self, method, endpoint, payload):
+        """ Make a request to Mercado Pago POS API.
+        :param method: "GET", "POST", ...
+        :param endpoint: The endpoint to be reached by the request.
+        :param payload: The payload of the request.
+        :return The JSON-formatted content of the response.
+        """
+        endpoint = MERCADO_PAGO_API_ENDPOINT + endpoint
+        header = {'Authorization': f"Bearer {self.mercado_pago_bearer_token}"}
+        try:
+            response = requests.request(method, endpoint, headers=header, json=payload, timeout=REQUEST_TIMEOUT)
+            return response.json()
```

Comment:
RequestException handle the HTTP-level errors, no?

Reply:
I need to forward the http error to the JS because in some cases (immediately canceling payment from the POS that is on the way to the terminal but not yet on it) Mercado Pago send a 409 error